README.md

```diff
@@ -36,10 +36,10 @@ Execute the following command to provision the tool:
 ansible-galaxy install -r requirements.yml -p galaxy
 
 # Install prerequisite infrastructure
-ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e include_tags=infrastructure
+ansible-playbook -i galaxy/openshift-toolkit/custom-dashboards/.applier galaxy/openshift-applier/playbooks/openshift-cluster-seed.yml -e include_tags=infrastructure  -e dashboard_namespace=tph2-monitoring
```

Comment:
@ramius345 is there a reason for this `tph2-monitoring` namespace separate from `custom-dashboards`?

Reply:
Just a mistake.  I was planning on updating the documentation to describe how to override the namespace.  I can just remove this for now.